### PR TITLE
Update connector tags

### PR DESCRIPTION
**Pull request by AathmanT**
As discussed in one of the meetings we decided to change the tags of the AI connectors

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]

File: openai.audio/Ballerina.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "speech-to-text", "whisper"]
+keywords = ["AI/Audio", "Cost/Paid", "speech-to-text", "whisper", "Vendor/OpenAI"]
```
Comment (nadheesh) [inline openai.audio/Ballerina.toml]: shouldn't whisper be "Whisper"?

File: openai.embeddings/Ballerina.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "OpenAI embeddings", "GPT-3 embeddings"]
+keywords = ["AI/Embeddings", "Cost/Paid", "OpenAI embeddings", "GPT-3 embeddings", "Vendor/OpenAI"]
```
Comment (nadheesh) [inline openai.embeddings/Ballerina.toml]: Do we need embeddings repeated in "AI/Embeddings", "OpenAI embeddings", "GPT-3 embeddings"?

File: openai.finetunes/Ballerina.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "fine-tune", "GPT-3 fine-tune"]
+keywords = ["AI/Fine-tunes", "Cost/Paid", "fine-tune", "GPT-3 fine-tune", "Vendor/OpenAI"]
```
Comment (nadheesh) [inline openai.finetunes/Ballerina.toml]: fine-tune repeated. is it required?

File: openai.images/Ballerina.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "image generation", "DALLE"]
+keywords = ["AI/Images", "Cost/Paid", "image generation", "DALLE", "Vendor/OpenAI"]
```
Comment (nadheesh) [inline openai.images/Ballerina.toml]: I think the keywords should not have repeats (e.g. image) 
Also DALLE is should be DALL-E or DALL-E 2

File: openai.text/Ballerina.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "Completions", "GPT-3", "Text completions"]
+keywords = ["AI/Text", "Cost/Paid", "Completions", "GPT-3", "Text completions", "Vendor/OpenAI"]
```
Comment (nadheesh) [inline openai.text/Ballerina.toml]: repeated "Text" and "completions"
can't we use GPT3.5?

Comment (AathmanT) [reply openai.text/Ballerina.toml]: Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/7d8940561515b3965ab48cc3c02f90e5a83a8627

File: openai.images/Ballerina.toml (hunk 4 above)
Comment (AathmanT) [reply openai.images/Ballerina.toml]: Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/7d8940561515b3965ab48cc3c02f90e5a83a8627

File: openai.finetunes/Ballerina.toml (hunk 3 above)
Comment (AathmanT) [reply openai.finetunes/Ballerina.toml]: Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/7d8940561515b3965ab48cc3c02f90e5a83a8627

File: openai.embeddings/Ballerina.toml (hunk 2 above)
Comment (AathmanT) [reply openai.embeddings/Ballerina.toml]: Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/7d8940561515b3965ab48cc3c02f90e5a83a8627

File: openai.audio/Ballerina.toml (hunk 1 above)
Comment (AathmanT) [reply openai.audio/Ballerina.toml]: Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/7d8940561515b3965ab48cc3c02f90e5a83a8627

File: openai.finetunes/Ballerina.toml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "fine-tune", "GPT-3 fine-tune"]
+keywords = ["AI/Fine-tunes", "OpenAI", "Cost/Paid", "Fine-tunes", "Files", "Models", "Vendor/OpenAI"]
```
Comment (nadheesh) [inline openai.finetunes/Ballerina.toml]: here do we need fine-tunes again?

**Review by nadheesh — APPROVED** [review]
looks good, check the comment and address accordingly

File: openai.finetunes/Ballerina.toml (hunk 6 above)
Comment (AathmanT) [reply openai.finetunes/Ballerina.toml]: Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/66ed081b167b621698f311731ba5058eae8cec1b

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**